sale_automatic_workflow_job/models/queue_job.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+
+
+class QueueJob(models.Model):
+    """ Job status and result """
+    _inherit = "queue.job"
+
+    def _related_action_sale_automaitc_workflow(self):
```

Comment:
```suggestion
    def _related_action_sale_automatic_workflow(self):
```

Typo in the name (to correct in the references too)